modules/Planner/planner_view_full.php

```diff
@@ -1312,7 +1330,13 @@
                             $form->addRow()->addSubmit()->addClass('mt-2');
                         }
 
-                        $page->addSidebarExtra($form->getOutput());
+                        // Hide roster for Students & Parents
+						$roleCat = $session->get('gibbonRoleIDCurrentCategory'); // 'Staff' | 'Student' | 'Parent' | etc.
```

Comment:
Thanks for the suggested change. In this case, rather than manually checking role categories, the built-in permission system would offer better control over view permissions.